recipes/freeglut/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class freeglutConan(ConanFile):
+    name = "freeglut"
+    description = "Open-source alternative to the OpenGL Utility Toolkit (GLUT) library"
+    topics = ("conan", "freeglut", "opengl", "gl", "glut", "utility", "toolkit", "graphics")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dcnieho/FreeGLUT"
```

Comment:
The github repo is a clone.
```suggestion
    homepage = "http://freeglut.sourceforge.net/"
```

Reply:
But it is official. I would prefer to keep it to GitHub

Reply:
Nope, according Github README, source forge is the official: https://github.com/dcnieho/FreeGLUT/#freeglut

Reply:
Yes and no I guess. The GitHub clone is from someone who is also the admin of the SourceForge project

Reply:
No objections from my side if you prefer to change it